app/main.py

```diff
@@ -1 +1,32 @@
-# write your code here
+import random
+import matplotlib.pylab as plt
+
+
+def flip_coin():
+    result = {key: 0 for key in range(11)}
+    for i in range(10000):
+        head = 0
+        for j in range(10):
+            side_of_coin = random.choice(["head", "tail"])
+            if side_of_coin == "head":
+                head += 1
+        result[head] += 1
+    final_result = {key: (value / 10000) * 100
+                    for key, value in result.items()}
+    return final_result
+
+
+coordinates = flip_coin()
```

Comment:
It's better to move this line into the `draw_gaussian_distribution_graph` function to avoid using global variables